site/docs/components/combo-box/examples.mdx

```diff
@@ -176,5 +176,13 @@ When the `value` prop is an object, the `valueToString` prop must be provided to
 
 Use the `truncate` prop when you are limited on space and want to prevent the multi-select combobox from spanning multiple lines.
 
+</LivePreview>
+
+<LivePreview componentName="combo-box" exampleName="ClearAll" >
```

Comment:
A question to ask the group, do we want to document this as clear all? Or an example of using a end adornment